packages/neos-ui-editors/src/Editors/Image/index.js

```diff
@@ -210,10 +210,14 @@ export default class ImageEditor extends Component {
     }
 
     handleChooseFromMedia = () => {
-        const {secondaryEditorsRegistry} = this.props;
+        console.log('options (r)', this.props.options)
```

Comment:
Forgot a log :)

Reply:
Doh, I forgot to run the linter, it's gone now. But I had to re-open the PR (=> #2758)